### PR TITLE
100% code coverage

### PR DESCRIPTION
**Pull request by Ian-Foote**

### CODE REVIEW

**Comment by Ian-Foote** [comment]
@KevinEtienne Review?

File: user_management/api/tests/test_views.py [hunk 1]
```diff
@@ -140,6 +140,13 @@ def test_delete_spacious_token(self):
         response = self.view_class.as_view()(request)
         self.assertEqual(response.status_code, status.HTTP_400_BAD_REQUEST)
 
+    def test_unknown_token(self):
+        """An unknown token is accepted silently."""
+        auth = 'Token unknown'
+        request = self.create_request('delete', HTTP_AUTHORIZATION=auth)
+        response = self.view_class.as_view()(request)
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
```
Comment (KevinEtienne) [inline user_management/api/tests/test_views.py]: I know this isn't the topic of this PR but do you think this is the intended behavior?

File: user_management/api/tests/test_throttling.py [hunk 2]
```diff
@@ -77,6 +77,13 @@ def test_user_auth_throttle_username(self):
         response = self.view_class.as_view()(request)
         self.assertEqual(response.status_code, self.throttle_expected_status)
 
+    def test_authenticated_user_not_throttled(self):
+        """An already authenticated user is not throttled."""
+        request = self.create_request('post', data={})
+        view = self.view_class.as_view()
+        response = view(request)
+        self.assertNotEqual(response.status_code, self.throttle_expected_status)
```
Comment (KevinEtienne) [inline user_management/api/tests/test_throttling.py]: Do you need to decorate the function with `@patch(THROTTLE_RATE_PATH, new={'logins': '1/minute'})` and call it two times to see if the authenticated user is not throttled?

File: user_management/api/tests/test_views.py (hunk 1 above)
Comment (meshy) [inline user_management/api/tests/test_views.py]: It seems sensible to me, but perhaps it should be rate-limited -- don't want people to be able to force a mass log out.

Comment (Ian-Foote) [inline user_management/api/tests/test_views.py]: The `GetAuthToken` view has throttles already: `UsernameLoginRateThrottle` and `LoginRateThrottle`.

File: Makefile [hunk 3]
```diff
@@ -10,5 +10,5 @@ release:
 
 test:
 	@coverage run ./user_management/tests/run.py
-	@coverage report --show-missing
+	@coverage report --show-missing --fail-under=100
```
Comment (meshy) [inline Makefile]: It doesn't look as though this is working.

Comment (meshy) [inline Makefile]: screenshot from 2015-02-27 10 04 12

Comment (Ian-Foote) [inline Makefile]: I needed to add it to `tox` too. fd89671

**Comment by Ian-Foote** [comment]
@KevinEtienne @meshy This can be merged now I think.

**Pull request merged**